examples/plugins/ceph.pp

```diff
@@ -1,7 +1,5 @@
 include collectd
 
 class { 'collectd::plugin::ceph':
-  osds          => [ 'osd.0', 'osd.1', 'osd.2'],
+  osds          => ['osd.0', 'osd.1', 'osd.2'],
```

Comment:
I thought the linter was also going to pick up the crazy about of extra whitespace here?

Reply:
I thought there was a different linter for this. I may be mistaken... Maybe it only verifies whether they are vertically aligned (which is always true when there is only one element)

Reply:
that should be fixed with this plugin: https://github.com/voxpupuli/puppet-collectd/pull/948